server/v2/api/grpc/nodeservice/service.go

```diff
@@ -0,0 +1,43 @@
+package nodeservice
+
+import (
+	context "context"
+	fmt "fmt"
+
+	"cosmossdk.io/core/server"
+)
+
+var _ ServiceServer = queryServer{}
+
+type queryServer struct {
+	cfg server.ConfigMap
+}
+
+func NewQueryServer(cfg server.ConfigMap) ServiceServer {
+	return queryServer{cfg: cfg}
+}
+
+func (s queryServer) Config(ctx context.Context, _ *ConfigRequest) (*ConfigResponse, error) {
```

Comment:
1 Q why we need v2 service since v1 was registed by comet. Like what is different?

Reply:
The comet one is here for backward compatibility. But we cannot get all data there.

Reply:
> The comet one is here for backward compatibility. But we cannot get all data there.

This service has less fields as the other aren't relevant as they are store server or consensus server dependent.